internal/recaptcha/pool.go

```
@@ -16,18 +16,36 @@ type TokenPool struct {
 			return FetchRecaptchaToken(ctx, net, proxyURI, cfg.DebugMode())
 		},
 		defaultProxy: cfg.ProxyURL,
-		routes: func() []string {
-			return sharedTokenRoutes(cfg)
-		},
+		routes:       func() []string { return sharedTokenRoutes(cfg) },
+	}
+	if cfg.RecaptchaTryEntryOrDirect() {
+		p.primary = func() []string {
+			out := []string{}
+			for _, e := range config.SelectEntryProxySequence(1, cfg) {
```

golangci-lint (2.12.2)

[error] S1011: should replace loop with out = append(out, config.SelectEntryProxySequence(1, cfg)...)

(staticcheck)